### PR TITLE
Exems by Humeniuk Katya IM-23

### PR DESCRIPTION
**Pull request by ekaterynah**

### CODE REVIEW

**Comment by tshemsedinov** [comment]
Please submit google form for exams

**Comment by ekaterynah** [comment]
> Please submit google form for exams

I've already submitted google form

**Comment by tshemsedinov** [comment]
Check it again I can't found your data

**Comment by ekaterynah** [comment]
> Check it again I can't found your data

Sent again, sorry

File: Tasks/day.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-let D = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+const parseDay = (day) => {
 
-_parse_day_ = (s) => {
-  let i;
-  for (i = 0; i < D.length; i++) {
-    if (s.startsWith(D[i].toLowerCase())) {
+  for (let i = 0; i < days.length; i++) {
+    const lowerDayName = days[i].toLowerCase();
+    if (day.startsWith(lowerDayName))
```
Comment (tshemsedinov) [inline Tasks/day.js:9]: Add code block or put next line in one line with if-statement

File: Tasks/password.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
 // Generate random password
-
-let GeneratePassword = (alphabet, length) => {
-  const MAX = alphabet.length;
-  key = '';
+'use strict'
+const generatePassword = (alphabet, length) => {
+  const max = alphabet.length;
+  let key = '';
```
Comment (tshemsedinov) [inline Tasks/password.js:5]: Naming hint: `password` instead `key`

**Review by tshemsedinov — APPROVED** [review]
LGTM (looks good to me)

**Pull request closed without merging**